.github/workflows/dependency-management.yml

```
@@ -112,7 +112,7 @@
               body: body,
               labels: ['security', 'dependencies', 'automated']
             });
-  
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)